src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsBrowserError.java

```diff
@@ -0,0 +1,47 @@
+package com.wikia.webdriver.testcases.adstests;
+
+import com.wikia.webdriver.common.core.Assertion;
+import com.wikia.webdriver.common.templates.TemplateNoFirstLoad;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.adsbase.AdsBaseObject;
+
+import org.openqa.selenium.JavascriptExecutor;
+import org.testng.annotations.Test;
+
+/**
+ * @ownership AdEng
+ */
+public class TestAdsBrowserError extends TemplateNoFirstLoad {
+
+  @Test(groups = "AdsScrollHandlerBrowserErrorMercury")
+  public void adsScrollHandlerBrowserErrorMercury() throws InterruptedException {
+    String testPage = urlBuilder.getUrlForPath("adtest", "Wikia_Ad_Testing");
+    testPage = urlBuilder.appendQueryStringToURL(testPage, "scrollhandler=1");
+    AdsBaseObject adsBaseObject = new AdsBaseObject(driver);
+    adsBaseObject.getUrl(testPage);
+
+    executeScript(addErrorListenerScript());
+    executeScript(scroll());
+
+    adsBaseObject.waitForJavaScriptTruthy("window.scrollY != 0");
+    Assertion.assertEquals(executeScript(getErrors()), "");
+  }
+
+  private String addErrorListenerScript() {
```

Comment:
I Must say the JavascriptActions class seems to be good destination for the method, as there is already isElementInViewPort method inside that class.